Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/activities/EmulationActivity.java

```diff
@@ -156,6 +165,14 @@ public final class EmulationActivity extends AppCompatActivity implements ThemeP
   public static final int MENU_ACTION_UNPAUSE_EMULATION = 33;
   public static final int MENU_ACTION_OVERLAY_CONTROLS = 34;
   public static final int MENU_ACTION_SETTINGS = 35;
+  public static final int MENU_ACTION_SKYLANDERS = 36;
+
+  private static Pair<Pair<Integer, Integer>, String> skylanderData;
```

Comment:
This data structure leaves the reader kind of confused as to what the values represent. Could you use a custom class here, either one that just contains the two ints or also the string (depending on which one you feel is more appropriate)?